### DISCUSSION TITLE
will it work for multivariate time series where values are mixture of continues and categorical values

### DISCUSSION THREAD

**Discussion by Sandy4321** [TDAmeritrade/stumpy#328] [Q&A]
great code thanks
may you clarify :
will it work for multivariate time series prediction both regression and classification

each row means the time series, and columns represent different continues or/and discrete/category observation.

1
where all values are continues values
weight height age target
time 1| 56 160 34 1.2
time 2| 77 170 54 3.5
time 3| 87 167 43 0.7
time 4| 55 198 72 0.5
time 5| 88 176 32 2.3
etc

2
or even will it work for multivariate time series where values are mixture of continues and categorical values
for example 2 dimensions have continues values and 3 dimensions are categorical values

		color        weight     gender  height  age      target 
time 1| black 56 m 160 34 yes
time 2| white 77 f 170 54 no
time 3| yellow 87 m 167 43 yes
time 4| white 55 m 198 72 no
time 5| white 88 f 176 32 yes

etc

**Comment by seanlaw** [comment]
> will it work for multivariate time series prediction both regression and classification

STUMPY computes a matrix profile and does not, in itself, build machine learning models. So, to answer your question, STUMPY can compute a multi-dimensional matrix profile and you can do whatever you want with it afterwards.

Additionally, regarding the type of data that can be used for computing the matrix profile, I'll kindly refer you to a similar question that you had raised around 2 years ago: 

https://github.com/TDAmeritrade/stumpy/discussions/60

Hopefully, that helps.
